test/disk_size_test.rb

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env rspec
+
+require_relative "spec_helper"
+require "storage/disk_size"
+require "pp"
+
+
+describe Yast::Storage::DiskSize do
+
+  describe "constructed empty" do
```

Comment:
usual rspec structure is 
```
describe Yast::Storage::DiskSize do
  describe ".new" do
      context "no parameter passed" do
        it "has size_k 0" do
        end
      end
      
    context "integer passed" do
       it "has size_k equal integer" do
         ....
       end
     end
  describe ".KiB" do
    ....
```

idea of rspec is to have specification that is tested. Try use `rspec --color --format documentation` and output should give you API specification.